fruit_classifier/predict/predict_utils.py

```diff
@@ -6,6 +6,51 @@
 from skimage.transform import resize
 
 
+def draw_text_with_background(output_image, text="text",
+                              text_offset_x=10, text_offset_y=25,
```

Comment:
PEP8 convention says that input parameters should be divided into separate lines:

output_image, 
text="text",
text_offset_x=10, 
text_offset_y=25,
...